sidhariyal/planet_names.fab

```diff
@@ -0,0 +1,10 @@
+# Ramesh Jayal Version 0.01

```

Comment:
Format clearly described in the User Guide, 9.3.3: 
the second column may contain the native spelling, optimally with English transliteration added. The third, "name" in `_("name")`, MUST use English characters (transliteration of the name if you want to give pronunciation help by international users, name or translation if translatable)  to allow translation. You may also provide a table in the description. To just translate planet names into Tamil for Tamil users, do that on Transifex.

Reply:
Noted with thanks, I'll incorporate it.